packages/commands/src/components/command-menu.js

```diff
@@ -271,7 +271,10 @@ export function CommandMenu() {
 			<div className="commands-command-menu__container">
 				<Command label={ inputLabel } onKeyDown={ onKeyDown }>
 					<div className="commands-command-menu__header">
-						<Icon icon={ inputIcon } />
+						<Icon
+							className="commands-command-menu__header-search-icon"
+							icon={ inputIcon }
+						/>
```

Comment:
<img width="679" height="446" alt="Command palette with flipped search icon" src="https://github.com/user-attachments/assets/312093ad-7bda-421d-85b7-cc1743211326" />

<img width="427" height="257" alt="Command palette with not-flipped search icon, RTL" src="https://github.com/user-attachments/assets/6f0fc655-a6ca-4a7d-8a44-f857971ad1d1" />